MANIFEST.in

```diff
@@ -1,3 +1,2 @@
-include pyproject.toml
-include *.md
-include LICENSE
+include CONTRIBUTING.md Makefile bin/*.py requirements/*
+recursive-include tests *.py
```

Comment:
Why? No one is repackaging this, so there's no reason to be including tests. And, this package is basically not complicated or dependent on external details, so there's no reason it'd break either.

Reply:
I'm not exactly sure about the intended use of source distributions but I usually expect them to deliver the full source tree, including tests and documentation.  setuptools-scm and flit take one step further and package everything that is tracked by VCS, but I think CI/CD configs are a bit too excessive to be manually included.

Reply:
Strong +1 to packaging everything needed to recreate the project in the source distribution.